src/main/java/net/dv8tion/jda/internal/handle/GuildRoleUpdateHandler.java

```diff
@@ -61,6 +61,8 @@ protected Long handleInternally(DataObject content)
         int color = rolejson.getInt("color");
         if (color == 0)
             color = Role.DEFAULT_COLOR_RAW;
+        DataObject colors = rolejson.getObject("colors");
```

Comment:
No need for a variable here, you will pass the `RoleColors` directly to the setter, not the `DataObject`